packages/mdc-dialog/mdc-dialog.scss

```diff
@@ -131,7 +133,6 @@ $mdc-dialog-dark-theme-bg-color: #303030 !default;
     padding: 8px;
   }
 
-  // TODO: Replace with breakpoint variable
```

Comment:
We would like to keep this comment in for now. `//` style comments are stripped out in the compiled CSS, so this only really exists for us in dev. These will all be removed once #23 lands.